lib/browser/browser.ts

```diff
@@ -210,48 +208,55 @@ class OktaAuthBrowser extends OktaAuthBase implements OktaAuth, SignoutAPI {
     this.authStateManager = new AuthStateManager(this);
   }
 
+  // TODO: deprecate in 5.0
   signIn(opts) {
-    const loginWithCredential = (opts) => {
-      opts = clone(opts || {});
-      const _postToTransaction = (options?) => {
-        delete opts.sendFingerprint;
-        return postToTransaction(this, '/api/v1/authn', opts, options);
-      };
-      if (!opts.sendFingerprint) {
-        return _postToTransaction();
-      }
-      return this.fingerprint()
-      .then(function(fingerprint) {
-        return _postToTransaction({
-          headers: {
-            'X-Device-Fingerprint': fingerprint
-          }
-        });
-      });
-    };
+    deprecate('This method will be deprecated in v5.0, please use signInWithCredentials() instead.');
```

Comment:
Should we say v5.0 of WHAT?  If this is buried in a stacktrace from minified code, I'd like to see the library and the actual method name listed.

Reply:
This is exactly what this `deprecated` function for, it adds prefix `[okta-auth-sdk] DEPRECATION: `. https://github.com/okta/okta-auth-js/blob/master/lib/util.ts#L239